src/packaging/metadata.rs

```diff
@@ -162,14 +166,6 @@ pub fn create_prefix_placeholder(
     };
 
     if file_mode == FileMode::Binary {
-        if prefix_detection.ignore_binary_files {
```

Comment:
I think it's still better to short cut this because we can save the expensive search (and rattler-build will be faster!).